script.js

```diff
@@ -128,6 +123,48 @@ restaurantsApp.showMore = function(result){
     });
 }
 
+restaurantsApp.handleHover = function() {
+    $('.imageWrapper').off('mouseover').on('mouseover', async function(e) {
```

Comment:
what does .off() do?

Reply:
if you are unfamiliar with .off() you can read about it here:
https://api.jquery.com/off/

we need to reset the event handlers every time more results are appended to the page. if we do not do this, another set of event handlers will be attached every time more results load